### DISCUSSION TITLE
Always Active doesn't appear to work for me?

### DISCUSSION THREAD

**Discussion by douglasg14b** [ActivityWatch/activitywatch#908] [Q&A]
My always active pattern: `Zoom Meeting|Google Meet|Microsoft Teams|Rest break|Micro-break|Workrave`

In any case where one of those windows are active and I meed the `afk` criteria for a lack of activity, it marks me as `afk.

Does this work for others? Ayy ideas why it doesn't appear to be working here?

![image](https://github.com/ActivityWatch/activitywatch/assets/1400380/2125e93a-b134-429b-befd-3fd9c8e26395)

![image](https://github.com/ActivityWatch/activitywatch/assets/1400380/4c8df57f-ffcc-4be6-acff-18e18c557b87)

**Comment by BelKed** [comment]
Active always pattern doesn't rewrite events in database from `afk` to `not-afk`. It only internally marks the events as `not-afk` while showing them in the visualisations or when counting the time you spent on a certain application.

**Reply by douglasg14b** [reply]
This doesn't appear to reflect in my activity though? My total active time discounts AFK time regardless of my Always Active settings, and increases if I manually change these afk times to be not-afk

**Reply by ErikBjare** [reply]
@douglasg14b It seems the "total active time" is a bit broken with respect to the "Always count as active" setting (as well as the "audible as active" setting), as it simply sums the durations of the non-afk events.

Can you check if the actual times for Zoom/Meet/Teams increase in the "Top window" visualization or such?

**Reply by douglasg14b** [reply]
They don, but only if Zoom is actually focused. Which it often is not when in an actual meeting

**Reply by BelKed** [reply]
ActivityWatch currently only tracks the focused window, so I assume this is the correct behaviour
